vignettes/cli-config-user.Rmd

```diff
@@ -9,18 +9,83 @@ editor_options:
     wrap: sentence
 ---
 
-These are environment variables and options that uses may set, to modify
+These are environment variables and options that users may set, to modify
 the behavior of cli.
 
-## User facing environment variables
+## Hyperlinks
+
+cli uses ANSI escape sequences to create hyperlinks.
+Specifically, cli creates [OSC 8 hyperlinks](https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda) that have this syntax:
+
+```
+OSC 8 ; {OPTIONAL PARAMS } ; {URI} ST {LINK TEXT} OSC 8 ; ; ST
+```
```

Comment:
To a certain extent, these docs are for me! I.e. I have learned a lot about OSC hyperlinks and how cli implements them and I want to store these notes somewhere in case I ever need to get back in here to tinker.